docs/build/smart-contracts/getting-started/hello-world-frontend.mdx

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 50
+sidebar_label: "5. Build a Hello World Frontend"
+title: "Build a frontend for the Hello World contract"
+description: "Build a frontend for the Hello World contract by using Stellar CLI to generate TypeScript bindings."
+---
+
+# 5. Build a Hello World Frontend
+
+In the previous examples we invoked the contracts using the Stellar CLI, and in this last part of the guide we'll create a web app that interacts with the Hello World contract through TypeScript bindings.
+
+:::info
+
+This example shows one way of creating a binding between a contract and a frontend, for a more comprehensive guide see the [Build a Dapp Frontend](../../apps/dapp-frontend.mdx) documentation.
+
+:::
+
+## Initialize a frontend toolchain
+
+You can build a Soroban app with any frontend toolchain or integrate it into any existing full-stack app. For this tutorial, we're going to use [Astro](https://astro.build/). Astro works with React, Vue, Svelte, any other UI library, or no UI library at all. In this tutorial, we're not using a UI library. The Soroban-specific parts of this tutorial will be similar no matter what frontend toolchain you use.
```

Comment:
Can we say Stellar app instead of Soroban app throughout the documentation? I think we will eventually move away from using "Soroban" as much as newcomers don't necessarily know what Soroban is, they just know Stellar.

Reply:
Done by using a standard Astro install instead of the template install